docs/java/sniffer.asciidoc

```diff
@@ -0,0 +1,126 @@
+== Sniffer
+
+Minimal library that allows to automatically discover nodes from a running
+Elasticsearch cluster and set them to an existing `RestClient` instance.
+It retrieves by default the nodes that belong to the cluster using the
+Nodes Info api and uses jackson to parse the obtained json response.
+
+Compatible with Elasticsearch 2.x and onwards.
+
+=== Maven Repository
+
+Here is how you can configure the dependency using maven as a dependency manager.
+Add the following to your `pom.xml` file:
+
+[source,xml]
+--------------------------------------------------
+<dependency>
+    <groupId>org.elasticsearch.client</groupId>
+    <artifactId>sniffer</artifactId>
+    <version>${es.version}</version>
```

Comment:
You say it is compatible with all versions of elasticsearch, so I'd imagine users replacing `{es.version}` with `1.7`.  Perhaps add a callout saying that this is the version of the ES client, starting with 5.0.0